src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/PolicyController.cs

```diff
@@ -120,5 +123,47 @@ public async Task<ActionResult> GetRolesWithAccess(string org, string app)
 
             return Ok(PolicyHelper.GetRolesWithAccess(policy));
         }
+
+        /// <summary>
+        /// Gets a list of resource policies for the list of org/apps
+        /// </summary>
+        /// <param name="orgAppList">The list of org/apps</param>
+        /// <returns>A list resourcePolicyResponses</returns>
+        [AllowAnonymous]
+        [HttpPost("/authorization/api/v1/resourcepolicies")]
```

Comment:
You can simplify the path:
```suggestion
        [HttpPost("resourcepolicies")]
```